scripts/script.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+class Person {
+  constructor(name, date, money) {
+    this.name = name;
+    this.birthday = date;
+    this.money = money;
+    this.accountHistory = [];
+    this.accountHistory.push({
+      name: 'Initial',
+      amountOfMoney: money
+    });
+  }
+
+  getInfo() {
```

Comment:
the function `getInfo` is responsible for return info about the person, that removes from function code with calculations age.